### DISCUSSION TITLE
Email me about hosting?

### DISCUSSION THREAD

**Discussion by davidkinetisense** [110yards/fantasy#82] [General]
david.schnare@gmail.com

**Comment by mdryden** [comment]
Thanks for the offer David, but the site is a cloud native application, so it's not easily moved to another hosting platform.  The back end database is Google Firestore, and the API is hosted in Google Cloud Run.

Part of the appeal of the project for me is the change to experiment with some different cloud architectural patterns; in my day job, I spend most of my time working with Windows servers and MS SQL.

I'm also pretty sure I can bring the hosting costs way down for next year - I made some mistakes in the event design, and the calculation stuff wound up triggering more page reloads and league calculations than it needed too.
